### DISCUSSION TITLE
Issue with Running plugNmeet on VPS Server alongside Moodle (Apache2 and HAProxy)

### DISCUSSION THREAD

**Discussion by salman99222** [mynaparrot/plugNmeet-server#594] [Q&A]
![IMG-20241111-WA0028](https://github.com/user-attachments/assets/8c19c4e4-45ca-4867-a0c5-511fbd3a6128)
Dear plugNmeet Community,

We are facing an issue when trying to run the plugNmeet application on our VPS server alongside Moodle. When we reach a certain point, as shown in the attached image, the application stops working and becomes unresponsive.

Our environment setup is as follows:

4-core processor

16 GB RAM

Dedicated IP

16 TB bandwidth

We have configured Docker to create separate containers for each application to avoid conflicts.

While this setup works fine on Amazon VPS, we are encountering issues with Hostinger VPS when using Apache2 and HAProxy.

Despite these resources and configurations, we are still facing difficulties in getting the application to function properly.

We kindly request your assistance in resolving this issue. Are there any additional settings or suggestions we can apply to resolve this?

Thank you for your cooperation and support.

Best regards,

**Comment by Saber-ebr** [comment]
I had a similar issue, was running nxproxy in order to set domains' A records alongside haproxy (while its used to run plugNmeet services).
I fixed it by using a third party service to set A records and ran haproxy only on the server.
